src/main/java/com/aws/greengrass/clientdevices/auth/connectivity/ConnectivityInformation.java

```diff
@@ -79,8 +82,6 @@ public List<ConnectivityInfo> getConnectivityInfo() {
             if (getConnectivityInfoResponse.hasConnectivityInfo()) {
                 // Filter out port and metadata since it is not needed
                 connectivityInfoList = getConnectivityInfoResponse.connectivityInfo();
-                cachedHostAddresses = new ArrayList<>(
```

Comment:
why aren't we doing anything here now?

Reply:
taken care of in recordConnectivityInformationForSource later on in this method